phpBB/includes/acp/acp_extensions.php

```diff
@@ -22,35 +22,37 @@
 class acp_extensions
 {
 	var $u_action;
+	var $tpl_name;
```

Comment:
To be really consistent we should only declare u_action as var :)
I will make another commit with this change.